[2024-06-19T10:20:00.353+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:20:00.368+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:40:00+00:00 [queued]>
[2024-06-19T10:20:00.374+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:40:00+00:00 [queued]>
[2024-06-19T10:20:00.374+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:20:00.392+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:40:00+00:00
[2024-06-19T10:20:00.394+0530] {standard_task_runner.py:63} INFO - Started process 4154 to run task
[2024-06-19T10:20:00.398+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:40:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmp5v10yh6a']
[2024-06-19T10:20:00.399+0530] {standard_task_runner.py:91} INFO - Job 119: Subtask backup_task
[2024-06-19T10:20:00.459+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:40:00+00:00 [running]> on host FN-21052948.
[2024-06-19T10:20:00.532+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:40:00+00:00'
[2024-06-19T10:20:00.533+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:20:00.541+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:20:00.542+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:20:00.546+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:20:00.553+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:20:00.559+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:20:00.572+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:20:00.788+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T10:20:00.788+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:20:00.788+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:20:00.789+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:20:00.819+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:40:00+00:00, execution_date=20240619T044000, start_date=20240619T045000, end_date=20240619T045000
[2024-06-19T10:20:00.853+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:20:00.875+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:20:00.882+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-19T10:24:26.416+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:24:26.434+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:40:00+00:00 [queued]>
[2024-06-19T10:24:26.442+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:40:00+00:00 [queued]>
[2024-06-19T10:24:26.442+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:24:26.458+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:40:00+00:00
[2024-06-19T10:24:26.460+0530] {standard_task_runner.py:63} INFO - Started process 4637 to run task
[2024-06-19T10:24:26.463+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:40:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpvjv2f9j9']
[2024-06-19T10:24:26.465+0530] {standard_task_runner.py:91} INFO - Job 124: Subtask backup_task
[2024-06-19T10:24:26.531+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:40:00+00:00 [running]> on host FN-21052948.
[2024-06-19T10:24:26.623+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:40:00+00:00'
[2024-06-19T10:24:26.623+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:24:26.631+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:24:26.631+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:24:26.639+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:24:26.647+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:24:26.654+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:24:26.670+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:24:26.898+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T10:24:26.898+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:24:26.898+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:24:26.899+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:24:26.931+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:40:00+00:00, execution_date=20240619T044000, start_date=20240619T045426, end_date=20240619T045426
[2024-06-19T10:24:26.958+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:24:26.977+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:24:26.988+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
